sonaranalyzer-dotnet/src/SonarAnalyzer.Common/SymbolicExecution/SymbolicValues/SymbolicValue.cs

```diff
@@ -82,7 +86,7 @@ public override string ToString()
 
         private static int SymbolicValueCounter = 0;
 
-        internal SymbolicValue()
+        public SymbolicValue()
             : this(SymbolicValueCounter++)

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this assignment of 'SymbolicValueCounter' or initialize it statically. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS3010)